src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,264 @@
+
+    # Wham:
+    col_name = get_col_name("wham", outlier_type)
+    outlier_wham_df = read_outlier(filename_wham, col_name, 11)
+    dict_wham = dict(list(zip(outlier_wham_df[ID_COL], outlier_wham_df[col_name])))
+
+    # merging all the dictionaries
+    outlier_dicts = [dict_manta, dict_melt, dict_wham]
+    merged_dicts = Counter()
+    for counted in outlier_dicts:
+        merged_dicts.update(counted)
+    all_outliers = dict(merged_dicts)
+    all_outliers_df = pd.DataFrame.from_dict(all_outliers, orient="index").reset_index()  # index_col=None
+    all_outliers_df.columns = [ID_COL, outlier_type + "_overall_outliers"]
+    return all_outliers_df
+
+
+def merge_evidence_qc_table(
+        filename_estimated_cn: str,
+        filename_mediancov: str,
+        filename_wgd: str,
+        filename_cnv_qvalues: str,
+        filename_high_manta: str,
+        filename_high_melt: str,
+        filename_high_wham: str,
+        filename_low_manta: str,
+        filename_low_melt: str,
+        filename_low_wham: str,
+        output_prefix: str) -> str:
+    """
+    Reads the provided TSV files (tab-delimited) and merges all the information in one table
+    serialized to the given output filename.
+    """
+    df_ploidy = read_ploidy(filename_estimated_cn)
+    df_bincov_median = read_bincov_median(filename_mediancov)
+    df_wgd_scores = read_wgd_scores(filename_wgd)
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"), 11)
+    df_total_high_outliers = read_all_outlier(filename_high_manta, filename_high_melt, filename_high_wham, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"), 11)
+    df_total_low_outliers = read_all_outlier(filename_low_manta, filename_low_melt, filename_low_wham, "low")
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers]
+    output_df = reduce(
+        lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs
+    ).fillna(0)  # "none"
+
+    # save the file
+    return output_df.to_csv(f"{output_prefix}.evidence_qc_table.tsv", sep="\t", header=True, index=False)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "estimated_copy_number_filename",
```

Comment:
Dashes are best practice for command-line arguments, rather than underscores. When you access these options later as variables the dashes will be automatically converted to underscores. \

I think since there are so many arguments they should be named rather than positional. You can add a short argument name if you want. 

Some of the files are marked optional in the WDL task - make sure that is handled on the script level.
```suggestion
        "--estimated-copy-number-filename",
```

Reply:
@epiercehoffman , this is a valid point, thanks for sharing. I also think that since these are required arguments, not optional, it might be better/easier if we go with underscore here. You can read more [here](https://stackoverflow.com/questions/12834785/having-options-in-argparse-with-a-dash)

Reply:
You can make them required but still use `--`. I think that for this quantity of arguments they shouldn't all be positional.

Reply:
@epiercehoffman , I think there could be positional for the python script, at the same time I agree that it's better to not have them as positional! Thanks for the comment. I applied your comment in [this](https://github.com/broadinstitute/gatk-sv/commit/0086235f678fb29d32c88aa9fdd15ba7a76a71b8) commit.

Reply:
The files have been set as optional.